include/taskolib/format.h

```diff
@@ -28,29 +28,44 @@
 #include <fmt/format.h>
 #include <sstream>
 
-#include "taskolib/time_types.h"
 #include "taskolib/Message.h"
+#include "taskolib/time_types.h"
+#include "taskolib/Timeout.h"
 
-template<> struct fmt::formatter<task::TimePoint> {
+using namespace std::literals;
+
+template<> struct fmt::formatter<task::Message> {
     constexpr auto parse(format_parse_context& ctx) -> decltype(ctx.begin()) {
         return ctx.begin();
     }
     template <typename FormatContext>
-    auto format(task::TimePoint const& val, FormatContext& ctx) -> decltype(ctx.out()) {
-        return format_to(ctx.out(), "{}", task::to_string(val));
+    auto format(task::Message const& val, FormatContext& ctx) -> decltype(ctx.out()) {
+        std::stringstream ss{ };
+        ss << val;
+        return format_to(ctx.out(), "{}", ss.str());
     }
 };
 
-template<> struct fmt::formatter<task::Message> {
+template<> struct fmt::formatter<task::Timeout> {
     constexpr auto parse(format_parse_context& ctx) -> decltype(ctx.begin()) {
         return ctx.begin();
     }
     template <typename FormatContext>
-    auto format(task::Message const& val, FormatContext& ctx) -> decltype(ctx.out()) {
+    auto format(task::Timeout const& val, FormatContext& ctx) -> decltype(ctx.out()) {
```

Comment:
Fine, but `task::Timeout` might be better passed by value – it is just an integer in disguise. The same goes for `TimePoint`.

Reply:
I guess it does not really matter one way or the other. 🤷